CareKitStore/CareKitStore/Structs/OCKScheduleElement.swift

```diff
@@ -88,7 +88,7 @@ public struct OCKScheduleElement: Codable, Equatable {
         }
     }
 
-    private let calendar: Calendar
+    private var calendar = Calendar.current
```

Comment:
#689 adds `calendar` to be a required encoded/decoded property which will break any `OCKTask` that's currently stored on a server in JSON because previous versions never contained this property.

Making the property a `var` and setting a default value will allow older saved OCKTasks's to be decoded as well as new ones (preserving all new updates with backwards compatibility). The other way would be to write a custom decoder for `OCKSheduleElement`, but that's more work and I don't think it's necessary.